app/pages/admin/featured-project-toggle.spec.js

```diff
@@ -29,8 +30,33 @@ function tearDown() {
 describe('Featured Project Toggle', function () {
   beforeEach(setup);
   afterEach(tearDown);
+
+  it('should render the featured checkbox when the project is lauch_approved', function () {
+    expect(checkbox).to.have.lengthOf(1);
+  });
+
+  it('should render a checked featured check box when the project is lauch_approved and featured', function () {
+    wrapper.setProps({
+      project: {
+        featured: true,
+        launch_approved: true
+      }
+    });
+    expect(checkbox).to.have.lengthOf(1);
```

Comment:
This doesn't test if the checkbox is checked.

Reply:
Oops sorry.